nbconvert/preprocessors/svg2pdf.py

```diff
@@ -64,8 +65,9 @@ def _inkscape_default(self):
         if sys.platform == "win32":
             wr_handle = winreg.ConnectRegistry(None, winreg.HKEY_LOCAL_MACHINE)
             try:
-                rkey = winreg.OpenKey(wr_handle, "SOFTWARE\\Classes\\inkscape.svg\\DefaultIcon")
+                rkey = winreg.OpenKey(wr_handle, "SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\App Paths\\inkscape.exe")
```

Comment:
Add a `rkey = winreg.OpenKey(wr_handle, "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\inkscape.exe")` to the except section and retry so we can fallback for older microsoft distros.

Reply:
Checking my system: my registry key is at:
```Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Classes\inkscape.svg\DefaultIcon```
So the original line is needed for me.